.github/workflows/post-jobs-slack.yaml

```
@@ -22,8 +22,12 @@ jobs:
           unique: "url"
 
           deploy: false
-          test: true
+          test: false
```

```suggestion
          test: true
```
And then can we do one more test run on your repo, then it should be good to go?

---

I think it worked:
https://github.com/crd477/jobs-updater/actions/runs/3797494327/jobs/6458485807
and results:
https://mastodon.sdf.org/@crd